scss/organisms/_banners.scss

```diff
@@ -58,6 +58,9 @@
 	position: relative;
 	box-sizing: border-box;
 }
+.jp-wpcom-connect__container > a:first-child {
```

Comment:
I had to change this to `.jp-wpcom-connect__inner-container > a:first-child {` for it to work.